src/screens/Asset/ReceiveModal.js

```diff
@@ -126,34 +180,61 @@ export default class ReceiveModal extends React.Component<Props, *> {
             />
           ),
         }]}
+        centerItem={
+          <ImageWrapper style={{ position: 'absolute', marginTop: -24 }}>
+            {!!profileImage && <ProfileImage
+              uri={`${profileImage}?t=${lastUpdateTime}`}
+              userName={username}
+              diameter={48}
+              borderWidth={0}
+              noShadow
+            />}
+            {!profileImage &&
+            <ProfileImagePlaceholder>
```

Comment:
Hm, `ProfileImage` handles placeholders by default if image is not preset.
See it's `DefaultPicture` component. If we need some changes in placeholders UI, maybe they are needed across the app? If not, maybe we can extend this component to accept placeholder's styling?